alacritty_terminal/src/tty/windows/conpty.rs

```diff
@@ -205,19 +203,11 @@ pub fn new<C>(config: &Config<C>, size: &SizeInfo, _window_id: Option<usize>) ->
         }
     }
 
-    // Get process commandline
-    let default_shell = &Shell::new("powershell");
-    let shell = config.shell.as_ref().unwrap_or(default_shell);
-    let mut cmdline = shell.args.clone();
-    cmdline.insert(0, shell.program.to_string());
-
-    // Warning, here be borrow hell
-    let cwd = config.working_directory().as_ref().map(|dir| canonicalize(dir).unwrap());
```

Comment:
As far as I can tell `dunce::canonicalize` exists for applications that do not handle the UNC paths that `std::fs::canonicalize()` creates.

The docs for [`CreateProcessW`](https://docs.microsoft.com/en-us/windows/win32/api/processthreadsapi/nf-processthreadsapi-createprocessw) specifically mention the working directory can be a UNC path. Which is why we can safely remove `dunce`.